src/frontend/components/Standings/components/DriverInfoRow/DriverInfoRow.tsx

```diff
@@ -54,28 +58,235 @@ export const DriverInfoRow = memo(({
+            {hidden ? null : (flairId && <CountryFlag
+              flairId={flairId}
+              size="sm"
+            />)}
+          </td>
+        ),
+      },
+      {
+        id: 'driverName',
+        shouldRender: (displayOrder ? displayOrder.includes('driverName') : true) && (config?.driverName?.enabled ?? true),
+        render: () => (
+          <td
+            key="driverName"
+            data-column="driverName"
+            className="px-2 py-0.5"
+            style={{ 
+              width: config?.driverName?.width ?? 250,
+              maxWidth: config?.driverName?.width ?? 250 
+            }}
+          >
+            <div className="flex items-center overflow-hidden">
+              <span
+                className={`animate-pulse transition-[width] duration-300 ${radioActive ? 'w-4 mr-1' : 'w-0 overflow-hidden'}`}
+              >
+                <SpeakerHighIcon className="mt-px" size={16} />
+              </span>
+              <div className="flex-1 overflow-hidden mask-[linear-gradient(90deg,#000_90%,transparent)]">
+                <span className="truncate">{hidden ? '' : name}</span>
+              </div>
+            </div>
+          </td>
+        ),
+      },
+      {
+        id: 'pitStatus',
+        shouldRender: (displayOrder ? displayOrder.includes('pitStatus') : true) && (config?.pitStatus ?? true),
+        render: () => (
+          <td key="pitStatus" data-column="pitStatus" className="px-1 text-center">
+            {hidden ? null : (onPitRoad && (
+              <span className="text-white animate-pulse text-xs border-yellow-500 border-2 rounded-md text-center text-nowrap px-2 leading-tight">
+                PIT
+              </span>
+            ))}
+          </td>
+        ),
+      },
+      {
+        id: 'carManufacturer',
+        shouldRender: (displayOrder ? displayOrder.includes('carManufacturer') : true) && (config?.carManufacturer?.enabled ?? true),
+        render: () => (
+          <td key="carManufacturer" data-column="carManufacturer" className="whitespace-nowrap">
+            <div className="flex items-center justify-center pr-2 text-center">
+              {hidden ? null : (carId && <CarManufacturer
+                carId={carId}
+                size="sm"
+              />)}
+            </div>
+          </td>
+        ),
+      },
+      {
+        id: 'badge',
+        shouldRender: (displayOrder ? displayOrder.includes('badge') : true) && (config?.badge?.enabled ?? true),
+        render: () => <td key="badge" data-column="badge" className="whitespace-nowrap text-center">{hidden ? null : badge}</td>,
+      },
+      {
+        id: 'iRating',
+        shouldRender: (displayOrder ? displayOrder.includes('iRating') : true) && (config?.iRating?.enabled ?? false),
+        render: () => <td key="iRating" data-column="iRating" className="px-2 text-center just whitespace-nowrap">{hidden ? null : iRating}</td>,
```

Comment:
```suggestion
        render: () => <td key="iRating" data-column="iRating" className="px-2 text-center whitespace-nowrap">{hidden ? null : iRating}</td>,
```
Not a valid css class - not sure if you intended it to be justify-center